dind/README.rst

```diff
@@ -0,0 +1,36 @@
+Small private cluster
+=====================
+
+Docker in Docker
+****************
+
+For devs and CI, this Docker in docker (dind) setup is targetted for sandboxes and CI.
+It aims to replace cluster_lab with Salt environnements
+
+* one dind instance holds one cluster
+* total of 4 dind instances: 4 clusters
```

Comment:
hum... as this repo is called cluster this is ambiguous !

I would say 1 cluster is composed with 4 nodes where this repo ``cluster`` is installed on each node !